packages/content-api/lib/index.js

```diff
@@ -13,7 +13,11 @@ export default function GhostContentAPI({url, host, ghostPath = 'ghost', version
     }
 
     if (this instanceof GhostContentAPI) {
-        return GhostContentAPI({url, version, key});
+        return GhostContentAPI({url, version, key, formatResponse});
+    }
+
+    if (!formatResponse) {
```

Comment:
I feel the formatResponse option should not throw an error if it isnt declared or assigned a value...
It should default to true just as mentioned here by the first person who commented on the issue [here](https://github.com/TryGhost/Ghost-SDK/issues/73#issuecomment-615893205)

Reply:
I'll look into it